### ISSUE TITLE
Refactor image attributes

### ISSUE THREAD

**Issue by javierm** [consul/consul#3153]
## Background

We've got several forms accepting image attachments. In all their controllers, there's a line like this one:

```ruby
image_attributes: [:id, :title, :attachment, :cached_attachment, :user_id, :_destroy]
```

If we ever change those image attributes (and it might happen when we switch to `ActiveSupport`, we'll need to change them in many places.

## Objectives

Define those attributes in only one place, and reuse them everywhere they're used.

**Comment by rogelio-o** [comment]
I can give a try if no one is working on it!

**Comment by microweb10** [comment]
Sure @rogelio-o! All yours ☺️.

**Comment by javierm** [comment]
Closed via #3170.
